curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md

```diff
@@ -12,12 +12,14 @@ Strategies with OAuth require you to have at least a <em>Client ID</em> and a <e
 Obtaining your <em>Client ID and Secret<em> from GitHub is done in your account profile settings under 'developer settings', then '<a href='https://github.com/settings/developers'>OAuth applications</a>'. Click 'Register a new application', name your app, paste in the url to your glitch homepage (<b>Not the project code's url</b>), and lastly for the callback url, paste in the same url as the homepage but with '/auth/github/callback' added on. This is where users will be redirected to for us to handle after authenticating on GitHub. Save the returned information as 'GITHUB_CLIENT_ID' and 'GITHUB_CLIENT_SECRET' in your .env file.
 On your remixed project, create 2 routes accepting GET requests: /auth/github and /auth/github/callback. The first should only call passport to authenticate 'github' and the second should call passport to authenticate 'github' with a failure redirect to '/' and then if that is successful redirect to '/profile' (similar to our last project).
 An example of how '/auth/github/callback' should look is similar to how we handled a normal login in our last project:
-<blockquote>
-app.route('/login')<br>
-&nbsp;&nbsp;.post(passport.authenticate('local', { failureRedirect: '/' }), (req,res) => {<br>
-&nbsp;&nbsp;&nbsp;&nbsp;res.redirect('/profile');<br>
-&nbsp;&nbsp;});
-</blockquote>
+
+```js
+app.route('/login')
+  .post(passport.authenticate('local', { failureRedirect: '/' }), (req,res) => {
+    res.redirect('/profile');
+  });
+```
```

Comment:
I was looking at this one... something isn't working quite right...
how does this, turn into this...
<img width="614" alt="Screen Shot 2019-05-09 at 9 32 18 AM" src="https://user-images.githubusercontent.com/20648924/57462054-c24e6580-723d-11e9-8377-e8fd08377c61.png">
no line breaks? 
I tried looking for spaces, adding spaces, changing things around - couldn't get it to display right

Reply:
@moT01 See my comment above to @ojeytonwilliams.

Reply:
That seems like a different problem - not sure though.

Reply:
@moT01 I see what you mean now.  @ojeytonwilliams My guess is these challenges use a different CSS file?